styles/globals.css

```diff
@@ -639,6 +639,38 @@ ul, ol {
   font-size: 20px;
 }
 
+#Sidemenu {
+  background-color: rgba(59, 71, 86, 0.9);
+}
+.menuLink {
+  display: inline-flex;
+  flex-direction: column;
+  justify-content: space-between; 
+  padding: 2px 12px 2px 0px;
+  color: black;
+  text-decoration: none;
+}
+.menuLink::after {
+  content: attr(data-text);
+  content: attr(data-text)/" ";

```

Comment:
Only one of these will get used -- the other will be overridden.

Reply:
i found this solution somewhere i am guessing there is a reason for it, maybe old/specific browser.